src/django/pfb_analysis/tasks.py

```diff
@@ -18,3 +18,7 @@ def create_batch_from_remote_shapefile(shapefile_url):
         batch.submit()
     finally:
         shutil.rmtree(tmpdir)
+
+
+def upload_local_analysis(analysis_url):
```

Comment:
This should take the `local_upload_task_uuid` since the job will need the entire task model to look up and use the related AnalysisJob and Neighborhood models as well as set Task status, etc.